### PR TITLE
Laravel mobile

### PR DESCRIPTION
**Pull request by baminou**

### CODE REVIEW

File: _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md [hunk 1]
```diff
@@ -13,4 +13,123 @@ tags:
...
+SSH to your host and make sure that you have the permissions to edit the directory where your website is going to be hosted. If you do not have the permissions,
+ask your server administrator to give you the permissions.
+
+Go to the Apache public directory (Usually /var/www/html, but might change depending on your apache setup)
+
+~~~BASH
+  cd /var/www/html
+~~~
+
+Create a new project using composer. If you do not have composer installed, I invite you to look at other solutions to create a new Laravel application [in their
```
Comment (Fgerthoffert) [inline _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md]: Could you detail what you mean by "look at other solutions". Something like this maybe "We are using composer for X reason, but it has the following limitations .... therefore we recommend looking at other solutions" ?

File: _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md [hunk 2]
```diff
@@ -13,4 +13,123 @@ tags:
...
+
+Change the permissions of the project folders
+
+~~~BASH
+sudo chmod –R 777 storage
+sudo chmod –R 777 bootstrap
+sudo chmod –R 777 public
+~~~
+
+Warning: The permissions of those directories are changed to 777 only for this tutorial. Do not use those permissions in production.
```
Comment (Fgerthoffert) [inline _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md]: Why, for the purpose of this tutorial are you using 777 ? Maybe add an explanation.

File: _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md [hunk 3]
```diff
@@ -13,4 +13,123 @@ tags:
 teaser:
     info: An easy way to create a mobile app from an existing Laravel application
 ---
-This tutorial describes an easy way to create a mobile app from your existing Laravel application.
+This tutorial describes an easy way to create a mobile app from your existing Laravel application. At the end of this tutorial, you will be able to create a hybrid app from an
```
Comment (Fgerthoffert) [inline _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md]: Maybe add a quick Intro on why using Laravel, versus other tools ? Added benefits or link to a page listing those ?

Comment (baminou) [reply _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md]: I did not want to talk too much about Laravel, but the post is answering a problem that I had using Laravel and that I was not able to find the explanation online. So I don't want to make any recommendations or comparisons to other tools as there is already a lot of different existing posts comparing the different frameworks.

**Comment by Fgerthoffert** [comment]
👍

**Review by Fgerthoffert — APPROVED** [review]

File: _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    - Android
+    - Mobile hybrid app
+    - PHP
+teaser:
+    info: An easy way to create a mobile app from an existing Laravel application
+---
+This tutorial describes an easy way to create a mobile app from your existing Laravel application. 
+
+Laravel is a free open-source PHP web Framework that helps you build web applications rapidly and efficiently using the model-view-controller (MVC) architectural pattern.
+To build a web application, Laravel is comparable to Django (Python) or Ruby on rails (Ruby). The choice of the right Framework to use for your next application if you are not
```
Comment (andricDu) [inline _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md]: rails --> Rails

File: _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+familiar with any of them would depends on how comfortable you feel with PHP, Python or Ruby. In this post, we are only going to talk about Laravel.
```
Comment (andricDu) [inline _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md]: ~would~

File: _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+At the end of this tutorial, you will be able to create a hybrid app from an existing Laravel project. A hybrid app have many advantages if you are familiar with HTML, CSS and Javascript:
```
Comment (andricDu) [inline _posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md]: have --> has

**Review by andricDu — CHANGES_REQUESTED** [review]

**Review by Fgerthoffert — APPROVED** [review]

**Review by Fgerthoffert — APPROVED** [review]

**Review by andricDu — APPROVED** [review]

**Pull request merged**